shhbt_config.yaml

```
@@ -429,3 +429,6 @@ signatures:
   - part:  'path'
     regex: '\.?mozilla[\\\/]firefox[\\\/]logins.json$'
     name: 'Firefox saved password collection (can be decrypted using keys4.db)'
+  - part: 'contents'
+    regex: "ghp_[0-9a-zA-Z]{36}"
```

@flippipe what do you think of adding all the remaining token types to the expression? Something like this:

```suggestion
    regex: "(ghp|gho|ghu|ghs|ghr)_[0-9a-zA-Z]{36}"
```

The idea is to catch one of those, since they are all used by GitHub.

Refs: https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/#identifiable-prefixes